assets/src/blocks/Articles/ArticlesEditor.js

```diff
@@ -1,162 +0,0 @@
-import {
-  CheckboxControl,
-  TextControl as BaseTextControl,
-  PanelBody,
-  Tooltip,
-} from '@wordpress/components';
-import {InspectorControls, RichText} from '@wordpress/block-editor';
-import withCharacterCounter from '../../components/withCharacterCounter/withCharacterCounter';
-import TagSelector from '../../components/TagSelector/TagSelector';
-import {PostSelector} from '../../components/PostSelector/PostSelector';
```

Comment:
If I'm not mistaken, this component isn't used anywhere else so it can be removed too?

Reply:
Done!